packages/terra-graphs-docs/src/terra-dev-site/ExampleGraphContainer/ExampleGraphContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ExampleGraphContainer.module.scss';
+
+const propTypes = {
+  /*
+   * The unique identifier for the graph container .
+   */
+  id: PropTypes.string.isRequired,
+};
+
+const ExampleGraphContainer = (props) => (
+  <React.Fragment>
+    <div id="tooltip" className="initial-tooltip" />
+    <div id={props.id} className="example-graph-container" />
+  </React.Fragment>
```

Comment:
More FYI than suggestion, but react fragments also have a shorthand that you can use.
```suggestion
  <>
    <div id="tooltip" className="initial-tooltip" />
    <div id={props.id} className="example-graph-container" />
  </>
```

Reply:
Updated syntax here: 7a87908